goto.html

```diff
@@ -7,7 +7,7 @@
     <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
     <link rel="stylesheet" href="./css/main.css" />
 
-    <script type="text/javascript" src="js/goto.js"></script>
+    <script type="module" src="js/goto.js"></script>
```

Comment:
any specific reason for importing as module?

Reply:
because i am importing some constants in the `goto.js` file.